marklogic-data-hub/src/test/java/com/marklogic/hub/core/HubConfigTest.java

```diff
@@ -80,4 +90,20 @@ public void testLoadBalancerProps() {
         assertEquals("host1", config.getLoadBalancerHosts()[0]);
     }
 
+    @Test
+    public void testHubInfo() {
+        HubConfig config = getHubConfig();
+
+        try {
+
+            System.out.println(config.getInfo());
```

Comment:
Do you feel like making any assertions in here?
We don't want to use System.out.println in uni tests.